torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -16,17 +16,10 @@ std::unordered_set<Symbol> skip_list = {
   prim::If,
   prim::Loop, //TODO: handle Loop
   prim::Print,
+  prim::RaiseException,
   prim::PythonOp, //may have side effects
   prim::LoadWorld,
   prim::StoreWorld,
-  //all the rand functions from native_functions.yaml
```

Comment:
Why are these deleted in this PR?

Reply:
I switched it to use the more comprehensive set of random ops